WordPress/openverse-frontend
# Fix horizontal scroll

## Fixes


Related to #710 by @zackkrida 

## Description


This PR attempts to fix the appearance of horizontal scrolling in the audio and image details pages on mobile. At first I thought the cause was just the logo text in the header but on the audio page the related audios were also causing the page to expand horizontally. I changed the related audios to use the 's' layout on mobile so they hide the waveform and fit in the page.

The current image page still has this problem even on desktop but it is fixed in #682, and the mobile view for the new page is fixed here then.

I also removed some classes from the default layout that were hiding the overflow, I'm not sure if this was necessary in order to the page look good on the iframe?

## Testing Instructions

Run the project and navigate to the audio details page on mobile viewport, see if there is any horizontal scroll. 

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
